core/node/da_dispatcher/README.md

```diff
@@ -0,0 +1,14 @@
+# DA dispatcher
+
+This crate contains an implementation of the DataAvailability dispatcher component, which sends a blobs of data to the
+corresponding DA layer.
```

Comment:
Please specify whether the component is a singleton. What are the implications of several component instances running in parallel?

Reply:
added some info about that